crates/typst-render/src/lib.rs

```diff
@@ -1023,8 +1023,9 @@ fn render_pattern_frame(state: &State, pattern: &Pattern) -> sk::Pixmap {
 }
 
 fn to_sk_color(color: Color) -> sk::Color {
-    let [r, g, b, a] = color.to_rgb().to_vec4_u8();
-    sk::Color::from_rgba8(r, g, b, a)
+    let [r, g, b, a] = color.to_rgb().to_vec4();
+    sk::Color::from_rgba(r, g, b, a)
+        .expect("components must always be in the range [0..=1]")
```

Comment:
I suggest using math notation

```suggestion
        .expect("components must always be in the range [0, 1]")
```

Reply:
I decided not to use it because it is slightly more ambiguous. Many languages use different interval notations after all. The notation I used is the one used by Rust and should thus be rather clear.

Reply:
Since it's not user facing, Rust notation is fine.